apps/python/web/routers/chat.py

```
@@ -103,6 +124,156 @@ def _drain_stderr() -> None:
+        "-p",
+        _build_skill_prompt(body),
+        "--permission-mode", "bypassPermissions",
+        "--output-format", "stream-json",
+        "--verbose",
+        "--add-dir", str(REPO_ROOT),
+        "--model", body.model,
+    ]
+    env = build_env(auth_mode=state_mod.read_state().auth_mode)
+
+    try:
+        result = subprocess.run(
```

Ruff (0.15.15)

[error] `subprocess` call: check for execution of untrusted input

(S603)